c++/src/capnp/rpc.h

```diff
@@ -202,6 +202,10 @@ class OutgoingRpcMessage {
   // Send the message, or at least put it in a queue to be sent later.  Note that the builder
   // returned by `getBody()` remains valid at least until the `OutgoingRpcMessage` is destroyed.
 
+  virtual void sendRealtime() = 0;
```

Comment:
Maybe this should have a default implementation which is `{ send(); }`? Otherwise landing this will break people's builds.